### DISCUSSION TITLE
How to remove applied filters from url

### DISCUSSION THREAD

**Discussion by SubhamKumar101201** [filamentphp/filament#10274] [Help]
how to remove the applied filters from url ,while apply table filters the url change with  tableFilters and tableSearch so is there any way to remove these  changes from default url

**Comment by SubhamKumar101201** [comment]
![Screenshot from 2023-12-13 10-05-45](https://github.com/filamentphp/filament/assets/88280454/008e444f-abef-421b-827e-3bd621882e5d)

**Comment by SubhamKumar101201** [comment]
![Screenshot from 2023-12-13 10-06-01](https://github.com/filamentphp/filament/assets/88280454/62a5a450-1727-4d54-a0da-d5fe36d62c47)

**Reply by m-pernkopf** [reply]
Do you want to delete them by action (like clicking an icon or similar) while being in the table where you applied your filter and search or don't you want them to be stored in the session (url), so they don't appear again, when you switched to another ressource and come back to your product-catalog again? If you don't want them to be stored (when switching between ressources) you could use shouldpersist functions and set them to false like described here: https://filamentphp.com/docs/2.x/tables/columns/getting-started#persist-search-in-session

M

**Comment by leandrocfe** [comment]
You can't do it I guess
